ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/PostFormVerifier.kt

```diff
@@ -175,8 +173,7 @@ class PostFormVerifier(private val response: Response, private val isRelayStateG
     @Suppress("ComplexMethod", "NestedBlockDepth")
     private fun verifyPostForm() {
         responseForm?.let {
-            if (it.hasNoAttributeWithNameAndValue(ACTION,
-                            checkNotNull(TestCommon.acsUrl[HTTP_POST]))) {
+            if (it.getAttribute(ACTION).isNullOrEmpty()) {
```

Comment:
❓ Why was the comparison to the`acsUrl` taken out?

Reply:
Because I'm effectively checking it now in the `CoreAuthnRequest` verifier. It's that weirdness where the check applies to both Core and Bindings.